<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>629</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>10</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>684</width>
     <height>550</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1120</width>
     <height>550</height>
    </size>
   </property>
   <widget class="QLabel" name="destDisplay">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="srcFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>661</width>
       <height>531</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="srcVerticalLayout" stretch="0,1,0">
      <item>
       <widget class="QLabel" name="srcHeader">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">width: 100%;</string>
        </property>
        <property name="text">
         <string>预览区域</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="srcDisplay">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="srcBottomhorizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="sizehorizontalLayout">
          <item>
           <widget class="QLabel" name="sizeLabel">
            <property name="text">
             <string>尺寸</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="sizeValue">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="scalehorizontalLayout">
          <item>
           <widget class="QLabel" name="scaleLabel">
            <property name="text">
             <string>缩放</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="scaleValue">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="collectionhorizontalLayout">
          <item>
           <widget class="QLabel" name="collectionLabel">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="text">
             <string>采集数量</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="collectionValue">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="fpshorizontalLayout">
          <item>
           <widget class="QLabel" name="fpsLabel">
            <property name="text">
             <string>帧频</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="fpsValue">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPreview"/>
   <addaction name="actionCollectSingle"/>
   <addaction name="actionCollectMulti"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionSetting"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionSpaceAlgo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPreview">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/play_32x32.png</normaloff>:/images/play_32x32.png</iconset>
   </property>
   <property name="text">
    <string>预览</string>
   </property>
   <property name="toolTip">
    <string>预览</string>
   </property>
  </action>
  <action name="actionCollectSingle">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/OneShot.png</normaloff>:/images/OneShot.png</iconset>
   </property>
   <property name="text">
    <string>单张采集</string>
   </property>
   <property name="toolTip">
    <string>单张采集</string>
   </property>
  </action>
  <action name="actionCollectMulti">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/ContinuousShot.png</normaloff>:/images/ContinuousShot.png</iconset>
   </property>
   <property name="text">
    <string>连续采集</string>
   </property>
   <property name="toolTip">
    <string>连续采集</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/settings_32x32.png</normaloff>:/images/settings_32x32.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/get_info_32x32.png</normaloff>:/images/get_info_32x32.png</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="toolTip">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/refresh_32x32.png</normaloff>:/images/refresh_32x32.png</iconset>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </action>
  <action name="actionSpaceAlgo">
   <property name="text">
    <string>空间算法</string>
   </property>
  </action>
  <action name="actionCloseAlgo">
   <property name="text">
    <string>关闭算法</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
